admin_guide/quota.adoc

```diff
@@ -476,3 +476,53 @@ If a quota has been defined for your project, see
 xref:../dev_guide/deployments/basic_deployment_operations.adoc#deployment-resources[Deployment Resources]
 for considerations on any deployment configurations.
 endif::[]
+
+ifdef::openshift-origin,openshift-enterprise,openshift-dedicated[]
+[[limited-resources-quota]]
+== Require explicit quota to consume a resource
+
+[NOTE]
+====
+This feature is tech preview and subject to change in future releases.
+====
+
+If a resource is not tracked by quota, a user has no restriction
+on the amount of resource that can be consumed.  For example,
+if there is no quota on storage related to the gold storage class,
+the amount of gold storage a user can create is unbounded.
```

Comment:
Quotas are placed on projects (right?). Should we refer to projects here, rather than "user"? 
_For example, if there is no quota on storage... the amount of golf storage a project can create is unbounded._